cli/tests/integrations/test_marathon_pod.py

```diff
@@ -121,9 +121,15 @@ def _assert_pod_remove(pod_id, extra_args):
     assert_command(cmd, returncode=0, stdout=b'', stderr=b'')
 
 
-def _assert_pod_show(pod_id, stdout):
+def _assert_pod_show(pod_id):
     cmd = _POD_SHOW_CMD + [pod_id]
-    assert_command(cmd, returncode=0, stdout=stdout, stderr=b'')
+    returncode, stdout, stderr = exec_command(cmd, env=None, stdin=None)
+
+    assert returncode == 0
+    assert stderr == b''
+
+    pod_json = json.loads(stdout.decode('utf-8'))
+    assert pod_json['id'] == util.normalize_marathon_id_path(pod_id)
 
```

Comment:
It won't, but the tests are still marked as skipped. I enabled them for local testing to make this change. Just trying to prevent a big change once we actually enable these tests.